src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AssembledBreakpoint.java

```diff
@@ -0,0 +1,160 @@
+ * of the genome (with one primary and two supplementary alignment records), there will be two AssembledBreakpoint
+ * objects created, one to represent each junction between alignment regions:
+ *
+ * Example Contig:
+ * ACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTG
+ * Alignment regions:
+ * |---------1:100-200------------|
+ *                                 |----------2:100-200------------------|
+ *                                                                       |----------3:100-200-----------------|
+ * Assmbled breakpoints:
+ * 1) links 1:100-200 to 2:100-200
+ * 2) links 2:100-200 to 3:100-200
+ *
+ * Inserted sequence contains portions of the contig that are aligned to neither region, and therefore may be inserted in
+ * the sample. For example, a translocation breakpoint with a microinsertion:
+ *
+ * Contig:
+ * ACTGACTGACTGACTGACTGACTGACTGACTGACTGACTGACTG
+ * Alignment regions:
+ * |-----1:100-200-------|
+ *                          |----2:100-200-----|
+ * Inserted sequence:
+ *  GA
+ *
+ * Homology represents ambiguity about the exact location of the breakpoint. For example, in this case one alignment
+ * region ends with "AC" and the next begins with AC, so we don't know if the AC truly belongs with the first or
+ * second alignment region.
+ *
+ * Contig:
+ * ACTGACTGACTGACTGACTGACTGACTGACTGACTGACTGACTG
+ * Alignment regions:
+ * |-----1:100-200-------|
+ *                    |-----2:100-200----------|
+ * Homology:
+ *  AC
+ */
+class AssembledBreakpoint {
+    String contigId;
+    AlignmentRegion region1;
+    AlignmentRegion region2;
+    String insertedSequence;
+    String homology;
+    List<String> insertionMappings;
+
+    public AssembledBreakpoint(final String contigId, final AlignmentRegion region1, final AlignmentRegion region2, final String insertedSequence, final String homology, final List<String> insertionMappings) {
+        this.contigId = contigId;
+        this.region1 = region1;
+        this.region2 = region2;
+        this.insertedSequence = insertedSequence;
+        this.homology = homology;
+        this.insertionMappings = insertionMappings;
+    }
+
+    @Override
+    public String toString() {
+        return contigId +
+                "\t" +
+                region1.toString() +
+                "\t" +
+                region2.toString() +
+                "\t" +
+                ("".equals(insertedSequence) ? "NA" : insertedSequence) +
+                "\t" +
+                ("".equals(homology) ? "NA" : homology);
+    }
+
+    /**
+     *  Parses a tab-delimited assembled breakpoint line into an AssembledBreakpoint object
+     */
+    public static AssembledBreakpoint fromString(String assembledBreakpointLine) {
+        final String[] fields = assembledBreakpointLine.split("\t");
+        return fromFields(fields);
+    }
+
+    public static AssembledBreakpoint fromFields(final String[] fields) {
```

Comment:
Again this was designed to read the output of `toString()`. Added some documentation.